modules/ROOT/pages/guided-setup.adoc

```diff
@@ -1,12 +1,17 @@
+
 = Guided Setup
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
+This page provides instructions for setting up API Community Manager using the guided setup option.
```

Comment:
Per the style guide we try to avoid making self-reference in the docs, what about something like: 

_Use the guided setup option to set up API Community Manager._ or 

_The guided setup assists you during the setup process of API Community Manager._

Reply:
Thanks. I will fix this.